src/CMakeLists.txt

```diff
@@ -439,6 +439,15 @@ if (TARGET ch_contrib::avrocpp)
     dbms_target_link_libraries(PRIVATE ch_contrib::avrocpp)
 endif ()
 
+if (TARGET ch_contrib::udns)
```

Comment:
Both new libraries are mandatory (i.e. their usage is not ifdef'd in ClickHouse's code), and as a result, we should link them here unconditionally.